docs/tableMean_current_sub_group.adoc

```diff
@@ -0,0 +1,112 @@
+ 3+|{attr}float :valid_range = -50.0, 50.0 
+
+ |{var}sample_v current_velocity_vessel_y(mean_time) |MA |Calculated vessel relative water column current velocity, in y direction.
+ 3+|{attr}:units = "m/s" 
+ 3+|{attr}:long_name = "Calculated mean water current velocity values relative own vessel, direction y"
+ 3+|{attr}float :valid_range = -50.0, 50.0 
+
+ |{var}sample_v current_velocity_vessel_z(mean_time) |MA |Calculated vessel relative water column current velocity, in z direction.
+ 3+|{attr}:units = "m/s" 
+ 3+|{attr}:long_name = "Calculated mean water current velocity values relative own vessel, direction z"
+ 3+|{attr}float :valid_range = -50.0, 50.0 
+
+ |{var}float mean_bin_lenght(mean_time) |MA |Distance covered by transmit pulse.
+ 3+|{attr}:units = "m" 
+ 3+|{attr}:long_name = "Distance covered by transmit pulse" 
+ 3+|{attr}float :valid_range = 0.0, 20.0 
+
+ |{var}double mean_platform_heading(mean_time) |MA |Calculated mean heading for the averaging period.
+ 3+|{attr}:units = "degrees_north" 
+ 3+|{attr}:long_name = "Calculated mean heading for the averaging period." 
+ 3+|{attr}float :valid_range = 0.0, 360.0 
+
+ |{var}double mean_platform_latitude(mean_time) |MA |Latitude representing the averaging period.
+ 3+|{attr}:units = "degrees_north" 
+ 3+|{attr}:long_name = "Latitude representing the averaging period." 
+ 3+|{attr}float :valid_range = -90.0, 90.0 
+
+ |{var}double mean_platform_longitude(mean_time) |MA |Longitude representing the averaging period.
+ 3+|{attr}:units = "degrees_east" 
+ 3+|{attr}:long_name = "Longitude representing the averaging period." 
+ 3+|{attr}float :valid_range = -180.0, 180.0 
+
+ |{var}double mean_platform_pitch(mean_time) |MA |Calculated mean pitch for the averaging period.
+ 3+|{attr}:units = "arc_degree" 
+ 3+|{attr}:long_name = "Calculated mean pitch for the averaging period." 
+ 3+|{attr}float :valid_range = -90.0, 90.0 
+
+ |{var}double mean_platform_roll(mean_time) |MA |Calculated mean roll for the averaging period.
+ 3+|{attr}:units = "arc_degree" 
+ 3+|{attr}:long_name = "Calculated mean roll for the averaging period." 
+ 3+|{attr}float :valid_range = -90.0, 90.0 
+
+ |{var}double mean_platform_vertical(mean_time) |MA |Calculated mean vertical for the averaging period.
+ 3+|{attr}:units = "m" 
+ 3+|{attr}:long_name = "Calculated mean vertical offset for the averaging period. Zero if velocity_depth_stabilisation is true." 
+
+ |{var}uint64 mean_time(mean_time) |MA |Timestamp at which each ping occurred.
+ 3+|{attr}:axis = "T" 
+ 3+|{attr}:calendar = "gregorian" 
+ 3+|{attr}:long_name = "TBD" 
+ 3+|{attr}:standard_name = "time" 
+ 3+|{attr}:units = "nanoseconds since 1601-01-01 00:00:00Z" 
+
+ |{var}float percent_good_limit |MA |Filtering parameter. Mean velocity value disgarded if quality below given limit.
+ 3+|{attr}:units = "%" 
+ 3+|{attr}:long_name = "Percent good limit used to filter the water current velocities" 
+ 3+|{attr}float :valid_range = 0.0, 100.0 
+
```

Comment:
This should be defined in ADCP_sub_group  as it is used first for ADCP_sub_group before averaging?